depends/Makefile

```diff
@@ -41,7 +41,10 @@ NO_SQLITE ?=
 NO_WALLET ?=
 NO_ZMQ ?=
 NO_USDT ?=
-MULTIPROCESS ?=
+# Default NO_MULTIPROCESS value is 1 on unsupported host platforms:
+# - Windows
+# - OpenBSB: https://github.com/capnproto/capnproto/pull/1907
```

Comment:
In commit "build: depends makes libmultiprocess by default" (48268facc9bb4f3e4f98cff5c21e1fe490b232e2)

OpenBSB typo